client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -59,27 +59,81 @@ export const filterBusinessExtensions = ( extensionInstallationOptions ) => {
 	);
 };
 
+const timeBoxes = [
+	{ name: '0-2s', max: 2 },
+	{ name: '3-4s', max: 5 },
+	{ name: '5-9s', max: 10 },
+	{ name: '10-15s', max: 15 },
+	{ name: '>15s' },
+];
+function getTimebox( timeInMs ) {
```

Comment:
A couple quick tests around this would be really awesome.

Reply:
Good point, I added them here: https://github.com/woocommerce/woocommerce-admin/pull/8042/commits/8ca96a069b8947a94f657c203e3039dc573e9e29